frontend/pages/hosts/details/cards/EmptyState/EmptyState.tsx

```diff
@@ -54,6 +56,20 @@ const EmptyState = ({ title, reason }: IEmptyStateProps): JSX.Element => {
           </div>
         </div>
       );
+    case "none-detected":
+      return (
+        <div className={`${baseClass} empty-${title}`}>
+          <div className={`${baseClass}__inner`}>
+            <div className={`${baseClass}__disabled`}>
+              <h1>No {formalTitle()} detected</h1>
+              <p>
+                {title === "munki-issues" &&
+                  "The last time Munki ran on this host, no issues were reported."}
+              </p>
+            </div>
+          </div>
+        </div>
```

Comment:
Feel we can find a common component between these error messages instead of html being repeated 4 times. only the messaging seems different between them. Maybe something with this API

```tsx
<EmptyHostMessage title={...} ...anyOtherProps >
  <p>...</p>
<EmptyHostMessage>
```